.existdb.json

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
I wouldn't remove this, it saves atom users some typing, and prevents accidental uploads of node_modules or bower_components which can be quite large

Reply:
I removed it because I had absolutely no idea what it was for. I can reinstate it...

Reply:
Still to have the old auto-syncing behaviour back, we ll probably need further changes to how atom tracks a local directory and then syncs with exist.